### PR TITLE
add logoutSSO 

### PR DESCRIPTION
**Pull request by taitelman**
when admin enabled login via SSO , this code show how to use the logout SSO . (simply invalidate SSO sessions in the client+server side)

### CODE REVIEW

File: lib/strategies/webapp-strategy.js [hunk 1]
```diff
@@ -528,4 +530,28 @@ function logAction(req, url, activity) {
...
+ * app.get("/logoutSSO", function(req, res, next) {
+ *	res.clearCookie("refreshToken");
+ *	webAppStrategy.logoutSSO(req,res, { successRedirect: "http://localhost:3000/niceGoodbyePage" });
+ * });
+ * @param req the http request
+ * @param res - the response object (will be used to redirect)
+ * @param options - need to contain successRedirect.
+ */
+WebAppStrategy.prototype.logoutSSO = function (req, res, options = {}) {
+	let redirectUri = options.successRedirect || '';
```
Comment (shiranyo) [inline lib/strategies/webapp-strategy.js]: I think we should not use the term "successRedirect" as it's very confusing. (in other places we use it as just the suffix without the context and here we should pass the full url)

File: lib/strategies/webapp-strategy.js [hunk 2]
```diff
@@ -528,4 +530,28 @@ function logAction(req, url, activity) {
...
+ * Calling the server SSO APIs will do nothing if you forgot to enable SSO in the server (tenant) configuration.
+ * Once you enabled SSO login in the server side you can try the logout in the client side:
+ * If you use the sample application , add a logout SSO UI widget , add to the app.js an express routing such as
+ * app.get("/logoutSSO", function(req, res, next) {
+ *	res.clearCookie("refreshToken");
+ *	webAppStrategy.logoutSSO(req,res, { successRedirect: "http://localhost:3000/niceGoodbyePage" });
+ * });
+ * @param req the http request
+ * @param res - the response object (will be used to redirect)
+ * @param options - need to contain successRedirect.
```
Comment (shiranyo) [inline lib/strategies/webapp-strategy.js]: Should we add the option to use all_apps? it's implemented
@motyd @taitelman

File: lib/strategies/webapp-strategy.js (hunk 1 above)
Comment (taitelman) [reply lib/strategies/webapp-strategy.js]: ok. changed the option JSON to be passed AS IS

File: lib/strategies/webapp-strategy.js (hunk 2 above)
Comment (taitelman) [reply lib/strategies/webapp-strategy.js]: if you add all_sessions to your options json , it will work now.

**Pull request merged**